### ISSUE TITLE
Vertical sidebar tabs sometimes become horizontal

### ISSUE THREAD

**Issue by aquamiau** [flandreas/antares#417]
I have not been able to find out the cause for this yet, but once it happens screen estate is lost and there doesn't seem to be a way to reset it other than restarting the app.

![unknown](https://user-images.githubusercontent.com/573744/177548072-470ed8d2-db2c-42d0-89f1-cc19a699f3a2.png)

**Comment by flandreas** [comment]
@aquamiau Yes, thank you for reporting this. I've experienced this problem a long time ago myself, but never again since then.

Can you guess what was causing this problem in your case? Any user interaction prior to it? Or did it occur right after starting the application?

**Comment by aquamiau** [comment]
Disconnecting or connecting a secondary screen seems to be one of the triggers, but I've also seen it happen without that.

**Comment by flandreas** [comment]
@aquamiau Tried to fix it by using the official style of implementing UIDelegates in Java. Before, it was kind of a hack.

I'm not sure though if this fixes it. Please report back if you see the problem occurring again after upgrading to 0.30.0.
